bin/run-test.sh

```diff
@@ -42,7 +42,10 @@ function run {
 run test test/hooks
 for test in test/standalone/test-*.js ;
 do
-  run "${test}"
+  if [[ ! $(node --version) =~ v0\.12\..* || ! "${test}" =~ .*trace\-koa\.js ]]
+  then
+    run "${test}"
+  fi
```

Comment:
Given that the second half makes it much longer, I am not sure this is a good idea anymore.